src/analysis/souffle/examples/multimic/multimic.arcs

```diff
@@ -0,0 +1,145 @@
+// Represents an actual microphone within a smart mic listening to audio.
+particle MicIn
+  audio: writes AudioData { }
+  claim audio is userSpeech
+
+// The permissions stubs below are not real devices. To allow guests to
+// influence the policy decisions of the smart mic, the smart mic is
+// representing the data gathered by the micrpohone as if it flowed through the
+// smartphones of users to allow their policies to attach tags to that data.
+//
+// Note: the permissions stubs add tags of the form userSpeechOfUserX. In
+// reality, this would likely be written more like the syntax that @aferr
+// proposed in PR #187, where tags can be refined by the user that introduced
+// them. The point here is that these stubs are adding a variant on the
+// userSpeech family of tags that imposes the policy preferences of the user
+// that they serve.
+particle UserAPermissionsStub
+  audio: reads writes AudioData { }
+  claim audio is userSpeechOfUserA
+
+particle UserBPermissionsStub
+  audio: reads writes AudioData { }
+  claim audio is userSpeechOfUserB
+
+particle UserCPermissionsStub
+  audio: reads writes AudioData { }
+  claim audio is userSpeechOfUserC
+
+particle StorageCredentialsSource
+  credentials: writes StorageCredentials { }
+  claim credentials is fullStoragePermissions
+
+particle DelegateStorePermissions
+  inPermissions: reads StoragePermissions { }
+  outPermissions: writes StoragePermissions { }
+  // The delegator claims that it downgrades the permissions to use storage
+  // from the full permissions to some less powerful delegated permissions.
+  claim outPermissions is not fullStoragePermissions and is delegatedPermissions
+  // Note: this is to halt if we are not allowed to downgrade permissions.
```

Comment:
This comment makes me think that there is some runtime enforcement happening here, but I thought the point of this PR was to show us how we don't need that. So I am confused.